cron-workflows/node-cpu-hog/workflow.yaml

```diff
@@ -50,6 +50,5 @@ spec:
                           - name: NODE_CPU_CORE
                             value: ""
       container:
-        image: lachlanevenson/k8s-kubectl
-        command: [sh, -c]
-        args: ['kubectl apply -f /tmp/chaosengine-1.yaml -n litmus']
+        image: litmuschaos/litmus-checker:latest
+        args: ["-file=/tmp/chaosengine.yaml","-saveName=/tmp/engine-name"]
```

Comment:
@gdsoumya is the saveName a mandatory arg? Can this be assumed w/ defaults

Reply:
It is not a mandatory arg, we can skip it. It is required for the generateName feature

Reply:
Should I remove it?

Reply:
Understood the context, we can keep it.